CHANGES.txt

```diff
@@ -60,6 +60,29 @@ Breaking Changes
 Changes
 =======
 
+- Changed primary key constraint of the
```

Comment:
Maybe we could squash the changes entry into one? Adds quite a bit of noise to the changes for the minor impact it has.

Reply:
Good point.